website/docs/reference/projects.md

```diff
@@ -3,131 +3,119 @@ id: projects
+The available project settings depend on a user's [root and project roles](/reference/rbac).
 
-1. In the top right of the project card, find the project menu represented by three vertical dots.
+## Archive a project
 
-![A list of projects. Each project has three vertical dots — a kebab menu — next to it.](/img/projects_menu_button.png)
+:::note Availability
 
-2. Click on Delete Project
+**Plan**: [Pro](https://www.getunleash.io/pricing) and [Enterprise](https://www.getunleash.io/pricing). | **Version**: 6.?+ in BETA.
 
-![A list of projects. Each project has three vertical dots — a kebab menu — next to it, which exposes a menu with the "Edit project" and "Delete project" options when interacted with.](/img/projects_delete_button.png)
+:::
 
-## Filter feature flags on projects {#filter-feature-toggles-on-projects}
+The archive projects functionality allows you to hide a project while maintaining historical project data. Archiving allows you to have short-lived projects, for example, to represent specific initiatives in your organization.
 
-When browsing the feature flags in Unleash, you might want to filter the view by looking only at the ones included in the project of interest. This is possible from the Feature flag overview.
+Before archiving a project, you must archive all feature flags within it. Once you archive a project, you cannot add or move feature flags to it.
 
-From the UI top navigation menu, choose "Feature flags".
+To archive a project:
 
-![The Unleash Admin UI navigation menu with the "Feature flags" option highlighted by a red arrow.](/img/projects_menu.png)
+1. Go to **Projects > Project settings > Settings**.
+2. Go to the **Archive project** section and click **Archive project**.
+3. Confirm by clicking **Yes, I'm sure**.
 
-The list of features flags can be filtered on the project of your choice. By default, all feature flags are listed in the view. You can use the search to filter to a specific project or even for multiple projects in the same time if you need.
+### Revive a project
 
-![The feature flag list with flags scoped to the "fintech" project. The filter is activated by using a form control.](/img/project_select.png)
+To revive an archived project, go to **Projects > Archived projects** and click **Revive project** on the project you want to restore.
 
-In the search you can type "project:specific-name" to filter that project only.
+## Delete a project
 
-![The feature flag list with an overlay listing all the projects available. You can select a project and the list will update with the flags belonging to that project.](/img/projects_select_dropdown.png)
+:::note Availability
 
-The view will now be updated with the filtered feature flags.
+**Plan**: [Pro](https://www.getunleash.io/pricing) and [Enterprise](https://www.getunleash.io/pricing).
 
-## Assigning project to a new feature flag {#assigning-project-to-a-new-feature-toggle}
```

Comment:
Removed this section because it's explained in https://docs.getunleash.io/reference/feature-toggles and anyways intuitive/self-explanatory.